docs/benchmarking.md

```diff
@@ -53,24 +162,31 @@ task jmh(type: JavaExec) {
     args '-rff', resultFile
 }
 ```
-Above is the script we use for our Cryptimeleon Benchmark project.
-It allows us to use certain JMH parameters in addition to just running all tests contained in the `jmh` source set.
+
+For example, to execute the previous JMH example (and only it), you would run 
+```bash
+./gradlew -q jmh -Pinclude="PS18VerifyBenchmark"
+```
+inside the folder of your Gradle project.
 
 # Group Operation Counting
 
-Cryptimeleon Math includes capabilities for group operation counting.
-Specifically, it allows for tracking group inversions, squarings, operations, as well as (multi-)exponentiations for a specific group.
+The collection of hardware-independent metrics such as group operations is implemented by the Cryptimeleon Math library.
+The main points of interest here are the `DebugBilinearGroup` and `DebugGroup` classes.
+The former allows for counting pairings, and the latter allows for counting group operations, group squarings (relevant for elliptic curves), group inversions, exponentiations, and multi-exponentiations.
+It is also able to track the number of times group elements have been serialized.
 
 ## DebugGroup
 
 The functionality of group operation counting is provided by using a special group, the `DebugGroup`.
+By simply using the `DebugGroup` to perform the computations, it automatically counts the operations done within it.
 
-*Note: Keep in mind that `DebugGroup` uses \\(\mathbb{Z}_n\\) under the hood, and so is only to be used when testing and/or counting group operations, not for other performance benchmarks.*
+*Note: Keep in mind that `DebugGroup` uses $$\mathbb{Z}_n$$ under the hood and is way faster than any secure group, and so is only to be used when testing and/or counting group operations, not for other performance benchmarks.*
 
 ```java
 import org.cryptimeleon.math.structures.groups.debug.DebugGroup;
 
-// instantiate the debug group with a name and its size
+// Instantiate the debug group with a name and its size
 DebugGroup debugGroup = new DebugGroup("DG1", 1000000);
```

Comment:
One remark here: to get useful results, the (debug) group size should be consistent with whatever (actual) group you want to count for. Otherwise, we get wrong results (e.g., if the group is small, also exponentiations take way fewer group ops).